chain/chain/src/chain.rs

```diff
@@ -4187,11 +4193,11 @@ impl<'a> ChainUpdate<'a> {
         Ok(())
     }
 
-    fn start_downloading_state(
+    fn get_state_dl_info(
```

Comment:
let's add a comment on what it does (and that it doesn't modify any state)

Reply:
Later I will be moving these functions to Chain instead of in ChainUpdate, hopefully that will make it more clear that this doesn't modify any state